keyboards/fungo/rev1/rev1.h

```diff
@@ -0,0 +1,79 @@
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "fungo.h"
+#include "quantum.h"
+
+
+#ifdef MASTER_RIGHT
+// 우측에  usb 케이블을 꽂아야 정상동작
+
+#define LAYOUT_fungo_split_num( \
+	L00, L01, L02, L03, L04, L05, L06,                       R06, R05, R04, R03, R02, R01, R00, \
+	L10, L11, L12, L13, L14, L15, L16,                       R16, R15, R14, R13, R12, R11, R10, \
+	L20, L21, L22, L23, L24, L25, L26,                       R26, R25, R24, R23, R22, R21, R20, \
+	L30, L31, L32, L33, L34, L35, L36, L46, L52,   R52, R46, R36, R35, R34, R33, R32, R31, R30, \
+	L50, L51, L40, L41,    L42,   L43, L44, L45,   R45, R44, R43,    R42,   R41, R40, R51, R50 \
+	) \
+	{ \
+		{ L00, L01, L02, L03, L04, L05, L06}, \
+		{ L10, L11, L12, L13, L14, L15, L16}, \
+		{ L20, L21, L22, L23, L24, L25, L26}, \
+		{ L30, L31, L32, L33, L34, L35, L36}, \
+		{ L40, L41, L42, L43, L44, L45, L46}, \
+		{ L50, L51, L52, KC_NO, KC_NO, KC_NO, KC_NO}, \
+		{ R06, R05, R04, R03, R02, R01, R00}, \
+		{ R16, R15, R14, R13, R12, R11, R10}, \
+		{ R26, R25, R24, R23, R22, R21, R20}, \
+		{ R36, R35, R34, R33, R32, R31, R30}, \
+		{ R46, R45, R44, R43, R42, R41, R40}, \
+		{ KC_NO, KC_NO, KC_NO, KC_NO, R52, R51, R50}, \
+	}
+#endif
+
+#ifdef MASTER_LEFT
+	#define LAYOUT_fungo_split_num( \
+	L00, L01, L02, L03, L04, L05, L06,                       R06, R05, R04, R03, R02, R01, R00, \
+	L10, L11, L12, L13, L14, L15, L16,                       R16, R15, R14, R13, R12, R11, R10, \
+	L20, L21, L22, L23, L24, L25, L26,                       R26, R25, R24, R23, R22, R21, R20, \
+	L30, L31, L32, L33, L34, L35, L36, L46, L52,   R52, R46, R36, R35, R34, R33, R32, R31, R30, \
+	L50, L51, L40, L41,    L42,   L43, L44, L45,   R45, R44, R43,    R42,   R41, R40, R51, R50 \
+	) \
+	{ \
+		{ L00, L01, L02, L03, L04, L05, L06}, \
+		{ L10, L11, L12, L13, L14, L15, L16}, \
+		{ L20, L21, L22, L23, L24, L25, L26}, \
+		{ L30, L31, L32, L33, L34, L35, L36}, \
+		{ L40, L41, L42, L43, L44, L45, L46}, \
+		{ L50, L51, L52, KC_NO, KC_NO, KC_NO, KC_NO}, \
+		{ R06, R05, R04, R03, R02, R01, R00}, \
+		{ R16, R15, R14, R13, R12, R11, R10}, \
+		{ R26, R25, R24, R23, R22, R21, R20}, \
+		{ R36, R35, R34, R33, R32, R31, R30}, \
+		{ R46, R45, R44, R43, R42, R41, R40}, \
+		{ KC_NO, KC_NO, KC_NO, KC_NO, R52, R51, R50}, \
+	}
+
+#endif
+
+
+
+
+
+
+
```

Comment:
There should not be any differentiation based on which side is master, because this can/will cause issues. 

Eg, what happens when somebody wants to set EE Hands?  Won't compile. Or split pin handedness. 

```suggestion
#define LAYOUT_fungo_split_num( \
	L00, L01, L02, L03, L04, L05, L06,                       R06, R05, R04, R03, R02, R01, R00, \
	L10, L11, L12, L13, L14, L15, L16,                       R16, R15, R14, R13, R12, R11, R10, \
	L20, L21, L22, L23, L24, L25, L26,                       R26, R25, R24, R23, R22, R21, R20, \
	L30, L31, L32, L33, L34, L35, L36, L46, L52,   R52, R46, R36, R35, R34, R33, R32, R31, R30, \
	L50, L51, L40, L41,    L42,   L43, L44, L45,   R45, R44, R43,    R42,   R41, R40, R51, R50 \
	) \
	{ \
		{ L00, L01, L02, L03, L04, L05, L06}, \
		{ L10, L11, L12, L13, L14, L15, L16}, \
		{ L20, L21, L22, L23, L24, L25, L26}, \
		{ L30, L31, L32, L33, L34, L35, L36}, \
		{ L40, L41, L42, L43, L44, L45, L46}, \
		{ L50, L51, L52, KC_NO, KC_NO, KC_NO, KC_NO}, \
		{ R06, R05, R04, R03, R02, R01, R00}, \
		{ R16, R15, R14, R13, R12, R11, R10}, \
		{ R26, R25, R24, R23, R22, R21, R20}, \
		{ R36, R35, R34, R33, R32, R31, R30}, \
		{ R46, R45, R44, R43, R42, R41, R40}, \
		{ KC_NO, KC_NO, KC_NO, KC_NO, R52, R51, R50}, \
	}
```

Reply:
This board has different circuits on the left and right. Therefore, the pin configuration is also different. The right keyboard does not use pins for spi communication ports for mouse connections. So if the master's position changes, the layout must also change.

Handedness by Pin
Handedness by Matrix Pin
Handedness by EEPROM

The above three methods do not know which layout to use. So I don't recommend the above method. It is recommended to use [Handiness by #define] instead.
Regarding the above, I corrected the wrong layout and modified the config.h file.
Please tell me in detail what to do if I did something wrong.

Reply:
Have you checked the layouts? They are exactly the same they don't specify any different positions between the 2 options.

> Handedness by Pin
> Handedness by Matrix Pin
> Handedness by EEPROM
>
> The above three methods do not know which layout to use. So I don't recommend the above method.

They should all use exactly the same layout this includes MASTER_LEFT and MASTER_RIGHT. There should be no difference.

> It is recommended to use [Handiness by #define] instead.

Where is this recommended? Both sides should be flashed with MASTER_LEFT or MASTER_RIGHT or whatever handedness option you pick. If you want to plug in either side and have it work correctly, one of the 3 handedness options you mention above will allow you to do so.

Reply:
Also, `MATRIX_COL_PINS_RIGHT` and `MATRIX_ROW_PINS_RIGHT` exist so that the left and right boards can use different pin outs.  This explicitly allows you to use different pins on the different halves.

https://docs.qmk.fm/#/feature_split_keyboard?id=hardware-configuration-options

Reply:
I revised the content 5 minutes ago.